src/api/foreground/foreground-setup-ar-protocol-links.ts

```diff
@@ -1,21 +1,17 @@
 import { sendMessage } from "@arconnect/webext-bridge";
-import type { PlasmoCSConfig } from "plasmo";
 import { isString } from "typed-assert";
 
-export const config: PlasmoCSConfig = {
-  matches: ["file://*/*", "http://*/*", "https://*/*"],
-  run_at: "document_start",
-  all_frames: true
-};
+// TODO: This is not enough for client-rendered pages (e.g. React apps):
```

Comment:
@7i7o See here. We might want to create a separate issue for that.

Reply:
Agree on separate issue 👍